src/js/components/stock-movement/common/ShipmentIdentifier.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiInformationLine } from 'react-icons/all';
+
+import ShipmentType from 'consts/shipmentType';
+
+const ShipmentIdentifier = ({ shipmentType, identifier }) => {
+  const getShipmentTypeIcon = () => ShipmentType[shipmentType?.enumKey ?? 'Default']?.icon ?? <RiInformationLine />;
```

Comment:
As I saw in the enum all of the shipmentTypes has their own icon, so what is the case for `?? <RiInformationLine />`?

Reply:
it is in case we add some additional `shipmentType` in the future and someone forgets to add it the icon to the enum, so to be sure that if we forget to provide an icon, it fallbacks to default's icon

Reply:
ok, so that's clear for me now